psi4/src/psi4/cc/ccdensity/kinetic.cc

```diff
@@ -117,6 +117,10 @@ void kinetic(std::shared_ptr<Wavefunction> wfn) {
     outfile->Printf("\t-V/T (corr)            = %20.15f\n", -vcorr / tcorr);
     outfile->Printf("\t-V/T (total)           = %20.15f\n", -vtot / ttot);
 
+    wfn->set_scalar_variable("CC CORRELATION KINETIC ENERGY", tcorr);
+    wfn->set_scalar_variable("CC CORRELATION POTENTIAL ENERGY", vcorr);
+    wfn->set_scalar_variable("CC CORRELATION VIRIAL RATIO", -vtot / ttot + vref / tref);
```

Comment:
The correlation energy virial ratio is -vcorr/tcorr, which isn't the same as what you've got here, unless I'm missing something due to lack of sleep.

Reply:
There's a definition question.

@loriab and I prefer defining the correlation virial ratio so that `reference + correlation = total`. That property is lost with your definition, but your definition is the one adopted by the current `cc` output. I can adopt either, so long as you and Lori agree on it. I'm not familiar enough with virial theorem literature to say whether either candidate definition is standard.

Reply:
I discussed this in [a paper in JCP](https://aip.scitation.org/doi/abs/10.1063/1.1535440) a while back, which is where I used these values as a measure of basis-set completeness for the correlated wave function only.  Like the total and Hartree-Fock energies, the ratio of the correlation contributions to the potential and kinetic energies also converges separately to -2, but it's not clear that what you're computing will do that.  Won't it converge to 0?  What values are you getting in your tests?

Reply:
Oh, I had no idea that you could use the correlation virial ratio for basis set completeness analysis. I've changed the variables and edited the documentation accordingly.